docs/reference/api/pytorch.txt

```diff
@@ -141,9 +140,9 @@ class with ``PyTorchTrial``, implement the following callback:
 .. code::
 
    class ReduceLROnPlateauEveryValidationStep(PyTorchCallback):
-       def __init__(self, context):
+       def __init__(self, optimizer):
```

Comment:
I have a slight preference for leaving this as `context` in order to hint users towards a more generally applicable practice.

Reply:
`context.get_optimizer()` will be removed. We recommend users to save their optimizers in member fields of their objects but they can't use it in the `Callback`.